src/Roslyn.Diagnostics.Analyzers/Core/DefaultableTypeShouldHaveDefaultableFieldsAnalyzer.cs

```diff
@@ -89,7 +89,9 @@ private static void AnalyzeField(SymbolAnalysisContext originalContext, IFieldSy
                 return;
             }
 
+#pragma warning disable RS1030 // Do not invoke Compilation.GetSemanticModel() method within a diagnostic analyzer
```

Comment:
Is it too complex to change now or do you prefer to handle this in a separate PR?

Reply:
Yeah, this applies to all symbol based analyzers.